packages/ui/src/primitives/tailwind/LoadingView/index.tsx

```diff
@@ -36,7 +36,8 @@ const LoadingView = ({
   fullSpace,
   containerClassname,
   titleClassname,
-  spinnerOnly
+  spinnerOnly,
+  delaySpinner = false
```

Comment:
I dont think we need this delay functionality to be in this control. It can stay in viewport file as this delay is a rare functionality And shouldnt be part of loadingView control

Reply:
I agree.